### PR TITLE
Zold wallets

### PR DESCRIPTION
**Pull request by g4s8**
#1350 - zold wallets in `people.xml`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1369](https://github.com/zerocracy/farm/issues/1369) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1369](https://github.com/zerocracy/farm/issues/1369) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1881242156); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/zerocracy/pmo/People.java [hunk 1]
```diff
@@ -414,6 +414,11 @@ public void wallet(final String uid, final String bank,
                 new Par("Litecoin address is not valid: `%s`").say(wallet)
             );
         }
+        if ("zld".equals(bank) && !wallet.matches("[a-z0-9\\-]{3,}")) {
```
[the remark this answers is not in the document]
Comment (g4s8) [reply src/main/java/com/zerocracy/pmo/People.java]: @krzyk we decided to use https://wts.zold.io for now, so the wallet is Github username here

**Comment by g4s8** [comment]
@krzyk fixed regex, please have a look

**Comment by krzyk** [comment]
@g4s8 OK, so in case of github username, the regex should be more complex, I don't know if there is an official, but the closes one is from https://github.com/shinnn/github-username-regex (javascript regexp syntax, it can be upper and lowercase):
```
/^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$/i
```

**Comment by g4s8** [comment]
@krzyk updated pattern, please look again

**Comment by krzyk** [comment]
@g4s8 thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1369](https://github.com/zerocracy/farm/issues/1369) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by ypshenychka** [comment]
@krzyk thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +30 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**